client/webserver/site/src/js/orderutil.ts

```diff
@@ -135,41 +137,41 @@ export function matchStatusString (m: Match) {
     // When revoked, match status is less important than pending action if still
     // active, or the outcome if inactive.
     if (m.active) {
-      if (m.redeem) return 'Revoked - Redemption Sent' // must require confirmation if active
+      if (m.redeem) return intl.prep(intl.ID_MATCH_STATUS_REVOKED, { status: intl.prep(intl.ID_MATCH_STATUS_REDEMPTION_SENT) }) // must require confirmation if active
       // If maker and we have not redeemed, waiting to refund, assuming it's not
       // revoked while waiting for confs on an unspent/unexpired taker swap.
-      if (m.side === Maker) return 'Revoked - Refund PENDING'
+      if (m.side === Maker) return intl.prep(intl.ID_MATCH_STATUS_REVOKED, { status: intl.prep(intl.ID_MATCH_STATUS_REFUND_PENDING) })
       // As taker, resolution depends on maker's actions while waiting to refund.
-      if (m.counterRedeem) return 'Revoked - Redeem PENDING' // this should be very brief if we see the maker's redeem
-      return 'Revoked - Refund PENDING' // may switch to redeem if maker redeems on the sly
+      if (m.counterRedeem) return intl.prep(intl.ID_MATCH_STATUS_REVOKED, { status: intl.prep(intl.ID_MATCH_STATUS_REDEEM_PENDING) }) // this should be very brief if we see the maker's redeem
+      return intl.prep(intl.ID_MATCH_STATUS_REVOKED, { status: intl.prep(intl.ID_MATCH_STATUS_REFUND_PENDING) }) // may switch to redeem if maker redeems on the sly
     }
     if (m.refund) {
-      return 'Revoked - Refunded'
+      return intl.prep(intl.ID_MATCH_STATUS_REVOKED, { status: intl.prep(intl.ID_MATCH_STATUS_REFUNDED) })
     }
     if (m.redeem) {
-      return 'Revoked - Redemption Confirmed'
+      return intl.prep(intl.ID_MATCH_STATUS_REVOKED, { status: intl.prep(intl.ID_MATCH_STATUS_REDEMPTION_CONFIRMED) })
     }
-    return 'Revoked - Complete' // i.e. we sent no swap
+    return intl.prep(intl.ID_MATCH_STATUS_REVOKED, { status: intl.prep(intl.ID_MATCH_STATUS_COMPLETE) }) // i.e. we sent no swap
```

Comment:
NBD, but the pattern repeats for all the revoked statuses, so I think this could all be more readable with a helper letting us do e.g. `return revokedMatchStatus(intl.ID_MATCH_STATUS_COMPLETE)`